instaclustr/structs.go

```diff
@@ -21,45 +21,46 @@ type OmitEmptyBool struct {
+	VPCType                       		string `json:"vpcType,omitempty" mapstructure:"vpc_type,omitempty"`
+	AWSAccessKeyId                		string `json:"aws.access.key.id,omitempty" mapstructure:"aws_access_key,omitempty"`
+	AWSSecretKey                  		string `json:"aws.secret.access.key,omitempty" mapstructure:"aws_secret_key,omitempty"`
+	S3BucketName                  		string `json:"s3.bucket.name,omitempty" mapstructure:"s3_bucket_name,omitempty"`
+	AzureStorageAccountName       		string `json:"azure.storage.account.name,omitempty" mapstructure:"azure_storage_account_name,omitempty"`
+	AzureStorageAccountKey        		string `json:"azure.storage.account.key,omitempty" mapstructure:"azure_storage_account_key,omitempty"`
+	AzureStorageContainerName     		string `json:"azure.storage.container.name,omitempty" mapstructure:"azure_storage_container_name,omitempty"`
+	SslEnabledProtocols           		string `json:"ssl.enabled.protocols,omitempty" mapstructure:"ssl_enabled_protocols,omitempty"`
+	SslTruststorePassword         		string `json:"ssl.truststore.password,omitempty" mapstructure:"ssl_truststore_password,omitempty"`
+	SslProtocol                   		string `json:"ssl.protocol,omitempty" mapstructure:"ssl_protocol,omitempty"`
+	SecurityProtocol              		string `json:"security.protocol,omitempty" mapstructure:"security_protocol,omitempty"`
+	SaslMechanism                 		string `json:"sasl.mechanism,omitempty" mapstructure:"sasl_mechanism,omitempty"`
+	SaslJaasConfig                		string `json:"sasl.jaas.config,omitempty" mapstructure:"sasl_jaas_config,omitempty"`
+	BootstrapServers              		string `json:"bootstrap.servers,omitempty" mapstructure:"bootstrap_servers,omitempty"`
+	Truststore                    		string `json:"truststore,omitempty" mapstructure:"truststore,omitempty"`
+	RedisMasterNodes              		int    `json:"masterNodes,omitempty" mapstructure:"master_nodes,omitempty"`
+	RedisReplicaNodes             		int    `json:"replicaNodes,omitempty" mapstructure:"replica_nodes,omitempty"`
+	RedisPasswordAuth             		*bool  `json:"passwordAuth,omitempty" mapstructure:"password_auth,omitempty"`
+	DedicatedZookeeper            		*bool  `json:"dedicatedZookeeper,omitempty" mapstructure:"dedicated_zookeeper,omitempty"`
+	ZookeeperNodeSize             		string `json:"zookeeperNodeSize,omitempty" mapstructure:"zookeeper_node_size,omitempty"`
+	ZookeeperNodeCount            		int    `json:"zookeeperNodeCount,omitempty" mapstructure:"zookeeper_node_count,omitempty"`
+	PostgresqlNodeCount           		int    `json:"postgresqlNodeCount,omitempty" mapstructure:"postgresql_node_count,omitempty"`
+	PostgresqlReplicationMode     		string `json:"replicationMode,omitempty" mapstructure:"replication_mode,omitempty"`
+	PostgresqlSynchronousModeStrict     *bool `json:"synchronousModeStrict,omitempty" mapstructure:"synchronous_mode_strict,omitempty"`
```

Comment:
Is this misaligned or is just github showing it as such

Reply:
I think Github?  In Intellij it looks aligned and I've run `terraform fmt`